.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,10 +1,15 @@
+---
+name: Bug Report
+about: File an issue about a bug
+title: "[BUG] "
```

Comment:
Is it possible to use labels instead of "tags"?

Reply:
Yes, but sometimes labels mean "this is a verified bug that needs to be fixed" rather than something that's just been reported. Of course, you could have more labels.

Reply:
That's a good point.

Reply:
Changed the others. This one is still open. `labels: [...]` would set it.

Reply:
> Yes, but sometimes labels mean "this is a verified bug that needs to be fixed" rather than something that's just been reported. Of course, you could have more labels.

I've been trying to do that, if I come across a bug I can confirm but don't have time to resolve.